api/api/management/commands/documentmedia.py

```diff
@@ -36,25 +41,54 @@ class FieldInfo:
     """Store common information that is applicable to all types of fields."""
 
     name: str
-    internal_type: str
+    field: type[Field]
     db_type: str
     is_relation: bool
-
-    notes: str | None = None
+    notes: str | None
 
     relation_info: RelationInfo | None = None
     value_info: ValueInfo | None = None
 
+    @staticmethod
+    def dj_docs_url(internal_type: str) -> str | None:
+        if internal_type == "ForeignObject":
+            # ``ForeignObject`` doesn't have any official documentation.
+            return None
+        if internal_type == "ArrayField":
+            # ``ArrayField`` is a PostgreSQL-specific field.
+            return "https://docs.djangoproject.com/en/dev/ref/contrib/postgres/fields/#arrayfield"
+        return f"https://docs.djangoproject.com/en/dev/ref/models/fields/#{internal_type.lower()}"
```

Comment:
I love this! 🤩 

Though, since we're typically not on a development version, we should probably link to stable instead:
```suggestion
            return "https://docs.djangoproject.com/en/stable/ref/contrib/postgres/fields/#arrayfield"
        return f"https://docs.djangoproject.com/en/stable/ref/models/fields/#{internal_type.lower()}"
```

Reply:
I didn't know there was a URL for stable. TIL! Updated in d479a59.